api/accounts_api_test.py

```diff
@@ -63,8 +63,11 @@ def test_create__normal_valid(self):
 
     new_appuser = (user_models.AppUser.query(
         user_models.AppUser.email == 'new_user@example.com').get())
-    self.assertEqual('new_user@example.com', new_appuser.email)
-    self.assertFalse(new_appuser.is_admin)
+    result_email = new_appuser.email
+    result_is_admin = new_appuser.is_admin
+    new_appuser.key.delete()
+    self.assertEqual('new_user@example.com', result_email)
+    self.assertFalse(result_is_admin)
```

Comment:
This change was added to make sure this user that is created for testing is deleted, as it was affecting unrelated tests written for these changes.